src/test/java/net/sf/log4jdbc/PreparedStatementSpyTest.java

```diff
@@ -1,28 +1,38 @@
 package net.sf.log4jdbc;
 
-import com.github.valfirst.slf4jtest.TestLoggerFactory;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-import uk.org.lidalia.slf4jext.Level;
+import static net.sf.log4jdbc.PreparedStatementSpyTest.Exec.withFreshlyLoaded;
+import static net.sf.log4jdbc.StatementSpy.SQL_SHOW_PARAMS_KEY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
 
+import com.github.valfirst.slf4jtest.TestLoggerFactory;
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
+import java.text.MessageFormat;
 import java.util.stream.Stream;
-
-import static net.sf.log4jdbc.PreparedStatementSpyTest.Exec.withFreshlyLoaded;
-import static net.sf.log4jdbc.StatementSpy.SQL_SHOW_PARAMS_KEY;
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.mock;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import uk.org.lidalia.slf4jext.Level;
 
 @DisplayName("Prepared Statement Spy should")
 public class PreparedStatementSpyTest {
 
+  private static final String INVALID_SQL_SHOW_PARAMS_VALUE = "moo"; // neither 'true', nor 'false'
```

Comment:
This value is a part of one of the test cases, it can't be reused. I personally like to have everything together, because narrower a scope, easier to grasp it. The same as `invalidShowSqlParamsLogMsg`.